common/appscale/common/ua_client.py

```diff
@@ -36,6 +36,23 @@ def add_admin_for_app(self, email, app_id):
     if response.lower() != 'true':
       raise UAException(response)
 
+  def commit_new_app(self, app_id, email, language):
+    """ Creates new project.
+
+    Args:
+      app_id: A string specifying an application ID.
+      email: A string specifying the user's email address.
+      language: A string specifying the project's language.
+    """
+    response = self.server.commit_new_app(app_id, email, language, self.secret)
+
+    # If the project already exists, consider the operation a success.
+    if response == 'Error: appname already exist':
```

Comment:
Could this be a constant that they both use so that if we ever want to change it we only have to change it in one place?